crates/ln-dlc-node/src/on_chain_wallet.rs

```diff
@@ -21,8 +21,9 @@ impl OnChainWallet {
         let data_dir = data_dir.join(&network.to_string());
         if !data_dir.exists() {
             // TODO: Had to create the `on_chain` directory manually for this to work in the tests
```

Comment:
nit: we could remove the comment then ;)

Reply:
Pushed as fixup - let me know when ready for merge and I can squash down